src/Sarif.Viewer.VisualStudio.UnitTests/CodeAnalysisResultManagerTests.cs

```diff
@@ -155,5 +155,83 @@ public void CodeAnalysisResultManager_GetRebaselinedFileName_IgnoresMismatchedFi
             Tuple<string, string>[] remappedPathPrefixes = target.GetRemappedPathPrefixes();
             remappedPathPrefixes.Should().BeEmpty();
         }
+
+        [Fact]
+        public void CodeAnalysisResultManager_GetRebaselinedFileName_WhenUserDoesNotSelectRebaselinedPath_UsesPathFromLogFile()
+        {
+            // Arrange.
+            const string FileNameInLogFile = @"C:\Code\sarif-sdk\src\Sarif\Notes.cs";
+
+            // The user does not select a file in the File Open dialog:
+            this.openFileDialogResult = false;
+
+            var target = new CodeAnalysisResultManager(
+                null,                               // This test never touches the file system.
+                this.keyboard,
+                this.commonDialogFactory);
+
+            // Act.
+            string rebaselinedFileName = target.GetRebaselinedFileName(uriBaseId: null, fileName: FileNameInLogFile);
+
+            // Assert.
+            rebaselinedFileName.Should().Be(FileNameInLogFile);
+
+            Tuple<string, string>[] remappedPathPrefixes = target.GetRemappedPathPrefixes();
+            remappedPathPrefixes.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void CodeAnalysisResultManager_GetRebaselinedFileName_WhenRebaselinedPathDiffersOnlyInDriveLetter_ReturnsRebaselinedPath()
+        {
+            // Arrange.
+            const string FileNameInLogFile = @"C:\Code\sarif-sdk\src\Sarif\Notes.cs";
+            const string RebaselinedFileName = @"D:\Code\sarif-sdk\src\Sarif\Notes.cs";
+
+            this.openFileDialogFileName = RebaselinedFileName;
+            this.openFileDialogResult = true;
+
+            var target = new CodeAnalysisResultManager(
+                null,                               // This test never touches the file system.
+                this.keyboard,
+                this.commonDialogFactory);
+
+            // Act.
+            string rebaselinedFileName = target.GetRebaselinedFileName(uriBaseId: null, fileName: FileNameInLogFile);
+
+            // Assert.
+            rebaselinedFileName.Should().Be(RebaselinedFileName);
```

Comment:
Names are really similar, sort of confusing. Not a huge issue, since it's just a small piece of code.

Reply:
@vinaykapadia  You're talking about the two variable names that differ only in case? Fair enough. I will change that variable name to `actualRebaselinedFileName` in all of the tests, and then merge. Thanks!